test.js

```diff
@@ -3,20 +3,24 @@ import difflet from 'difflet';
 import m from './';
 
 const diff = difflet({indent: 2});
-
-test.cb(t => {
-	const expected = {
-		'elements/colors': {
-			style: true,
-			documentation: true
-		},
-		'elements/grid': {
-			style: true,
-			documentation: true
+const expected = {
+	elements: {
+		colors: {
+			path: 'src/elements/colors/',
```

Comment:
My logic is that those as well as the folders passed in the config should be relative to the base path. './fixtures/simple' in this example